packages/cli/smoke-test/openapi/real-world-apis.js

```diff
@@ -0,0 +1,74 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const {supertest, TestSandbox} = require('@loopback/testlab');
+const path = require('path');
+const assert = require('yeoman-assert');
+const generator = path.join(__dirname, '../../generators/openapi');
+
+// Test Sandbox
+const SANDBOX_PATH = path.resolve(__dirname, '../.sandbox');
+const sandbox = new TestSandbox(SANDBOX_PATH);
+const testUtils = require('../../test/test-utils');
+
+describe('Real-world APIs', () => {
```

Comment:
What's the purpose of this test file? It's not going to be run as part of our regular test suite, will it?

Please add some sort of documentation for people maintaining/contributing to `openapi` generator and explain how to use this test file, etc.